B.py

```diff
@@ -0,0 +1,2 @@
+def hallo():
+   print("こんにちは”)
```

Comment:
```suggestion
   print("こんにちは")
```
ダブルコーテーションを修正してください。